coreos-base/oem-interoute/files/cloud-config.yml

```diff
@@ -0,0 +1,49 @@
+#cloud-config
+
+coreos:
+    units:
+      - name: vmtoolsd.service
+        command: start
+        content: |
+          [Unit]
+          Description=VMware Tools Agent
+          Documentation=http://open-vm-tools.sourceforge.net/
+          ConditionVirtualization=vmware
+
+          [Service]
+          ExecStartPre=/usr/bin/ln -sfT /usr/share/oem/vmware-tools /etc/vmware-tools
```

Comment:
vmware-tools needs that path and cloudinit runs too late to do this with tmpfiles, iirc it is this way in the existing vmware OEM